pkg/datapath/linux/ipsec.go

```diff
@@ -35,6 +35,10 @@ func (n *linuxNodeHandler) getDefaultEncryptionInterface() string {
 	if len(option.Config.EncryptInterface) > 0 {
 		iface = option.Config.EncryptInterface[0]
 	}
+	devices := option.Config.GetDevices()
+	if len(devices) > 0 {
+		iface = devices[0]
+	}
```

Comment:
The commit message says:

```
 If no encryption interface is given (i.e., --encrypt-interface),
then we fallback to using --devices if it's given.
```

But the logic here is using --devices as default, it's --device is not given then we fallback to --encrypt-interface, right?

Reply:
The commit message is what is wrong. The code was correct but convoluted so I tried to simplify it. Would appreciate a second review on that function :-)